src/components/DeeplinkWrapper/index.website.js

```diff
@@ -54,11 +54,7 @@ class DeeplinkWrapper extends PureComponent {
         const expensifyUrl = new URL(CONFIG.EXPENSIFY.NEW_EXPENSIFY_URL);
         const params = new URLSearchParams();
         params.set('exitTo', `${window.location.pathname}${window.location.search}${window.location.hash}`);
-        if (!this.props.session.authToken) {
```

Comment:
Removing this check since this component is only ever rendered in an authenticated context now.